Source/CarthageKit/GitHub.swift

```diff
@@ -306,8 +304,10 @@ extension GitHubRelease: CustomStringConvertible {
 }
 
 extension GitHubRelease: Decodable {
-	public static func create(ID: Int)(name: String?)(tag: String)(draft: Bool)(prerelease: Bool)(assets: [Asset]) -> GitHubRelease {
-		return self.init(ID: ID, name: name, tag: tag, draft: draft, prerelease: prerelease, assets: assets)
+	public static func create(ID: Int) -> String? -> String -> Bool -> Bool -> [Asset] -> GitHubRelease {
+		return { name in { tag in { draft in { prerelease in { assets in
+			return self.init(ID: ID, name: name, tag: tag, draft: draft, prerelease: prerelease, assets: assets)
+		} } } } }
```

Comment:
fwiw, these `create` functions shouldn't be necessary anymore. If you use something like [Curry.framework][curry], you could use `curry(GitHubRelease.init)` instead. These `create` functions were only really necessary pre-swift 2.0.

[curry]: https://github.com/thoughtbot/Curry